### ISSUE TITLE
W41 Lib Team Call

### ISSUE THREAD

**Issue by nondiremanuel** [bitcoindevkit/.github#80]
:calendar: Tues October 10, 2023 12:00 UTC
:round_pushpin: Discord General Voice Channel

**Please add comments to this issue to suggest issues and PRs to discuss.**

## General Discussion Items

- [ ] BDK 1.0 milestones and timeline
- [ ] BDK Review Club
- [ ] Any review begs? :pray:
- [ ] Blockers? 🪨

#### Recently merged 🎉:
- [x] bitcoindevkit/bdk#1064
- [x] bitcoindevkit/bdk#1041
- [x] bitcoindevkit/bdk#1110: audit job failed after merge
#### Need Review:
- [x] bitcoindevkit/bdk#1124: examples for rpc -> closed since already included in #1041
- [x] bitcoindevkit/bdk#1163: just added this bug fix to electrum sync. Can we squeeze it into alpha.2?

#### To Do:
- [x] bitcoindevkit/bdk#1162: on CI -> not necessarely linked to alpha but annoying. For the moment assigned to alpha.2
- [x] bitcoindevkit/bdk#1138: related to #1064, small change -> since it's very small and good to have early, moved to alpha.2

#### Need Review:
- [x] bitcoindevkit/bdk#1145: not clear if it fixes the syncing issue. Replaced by #1163. We will close this PR when that one is merged
- [x] bitcoindevkit/bdk#1116 -> needs further review after last changes.  Different approach was chosen, needs to be substituted by a new PR. The milestone will depend on the timing of the rework -> moved to beta milestone for the moment but it can be moved in the future
- [x] bitcoindevkit/bdk#1117 -> the author is waiting for some explanation
- [x] bitcoindevkit/bdk#1109 -> keep in alpha.3
#### In progress:
- [x] bitcoindevkit/bdk#1028: needs to be rebased after alpha.2 milestone
- [x] bitcoindevkit/bdk#1115: on hold, needs some comment from evan and/or Lloyd -> the old method has some problems but it's not clear the impact of it. Degree of complexity (and consequently milestone) needs to be discussed
- [x] bitcoindevkit/bdk#1106 -> important to have but okay to leave it in alpha.3
- [x] bitcoindevkit/bdk#1128: another approach has to be tried
- [x] bitcoindevkit/bdk#1141: either someone take this over or it will be low priority for the original author (junderw) -> important to keep it since it affects the Wallet API. Not clear if it is close to end or not
#### To Do:
- [x] bitcoindevkit/bdk#1144: new bug
- [x] bitcoindevkit/bdk#1142: new bug to be investigated before alpha.3 and then evaluate the priority
- [x] bitcoindevkit/bdk#1101 -> ongoing discussion
- [x] bitcoindevkit/bdk#1042
- [x] bitcoindevkit/bdk#1079: Lloyd needs some clarification on the API -> needs further discussion
- [x] bitcoindevkit/bdk#1113: can we move tests to beta (then if they are done before we can include them in alpha.3)? -> ok to move to beta
- [x] Documentation of methods can be included sooner, but a review will be needed in Beta -> LaggingTimes will have a look at this and #1094 (tests in general)
- [x] bitcoindevkit/bdk#1094: to be done after #1113 -> if we run out of time we will move it to the Beta milestone
- [x] bitcoindevkit/bdk#1127
- [x] bitcoindevkit/bdk#1153: new issue (see #1139 below)
- [x] bitcoindevkit/bdk#1154: new issue

### Milestone TBD:

- [x] bitcoindevkit/rust-esplora-client#59: issue originally in 1.1 version, proposal of a quick fix to fit it into beta of 1.0 -> moved to next esplora release
- [x] bitcoindevkit/bdk#1139: discussion ongoing on the concept. Related to #1153 issue that was assigned to alpha.3
- [x] bitcoindevkit/bdk#1134
- [x] bitcoindevkit/bdk#979: related to 1145 (alpha.3). Should it be included?

## Top Maintenance PRs and issues

0.29 version released!!! 🎉

## New Projects and Support
- [x] bitcoindevkit/bdk#1147: discussion -> not too urgent, put in the Beta at the moment
- [x] Roadmap blog post

## Upcoming Events

- [x] [BTC Azores](https://btcazores.com/) Sept 23-24
- [ ] [PlanB Lugano](https://planb.lugano.ch/?lang=it) Oct 20-21

🌅 *[Future Call Notes](https://github.com/bitcoindevkit/.github/issues?q=label%3ACall+is%3Aopen)*

📜 *[Past Call Notes](https://github.com/bitcoindevkit/.github/issues?q=label%3ACall+is%3Aclosed)*

[BDK Milestones]:https://github.com/bitcoindevkit/bdk/milestones
[BDK Core Staging Roadmap]:https://github.com/users/LLFourn/projects/2
[BDK 1.0]:https://github.com/orgs/bitcoindevkit/projects/14
[BDK Maintenance]: https://github.com/orgs/bitcoindevkit/projects/6
[BDK-CLI Roadmap]:https://github.com/orgs/bitcoindevkit/projects/7
[Documentation Roadmap]:https://github.com/orgs/bitcoindevkit/projects/9
[BDK Core Project]:https://github.com/users/LLFourn/projects/2/views/1
